### PR TITLE
Module 13

### PR DESCRIPTION
**Pull request by sergei820**
Module 13 task is done, please, check when you have time:
module_13

### CODE REVIEW

File: module_13/driver/chromedriver-win64/chromedriver.exe
Comment (VAlexandrov911) [inline module_13/driver/chromedriver-win64/chromedriver.exe:1]: Place the chromedriver.exe's path into PATH (via Environment Variable)
you can find more info with screenshots here for example
https://www.browserstack.com/guide/run-selenium-tests-using-selenium-chromedriver

File: module_13/tests/base_test.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+from selenium import webdriver
+
+
+class BaseTest:
+    @classmethod
+    def setup(cls):
+        chrome_driver_path = "driver/chromedriver-win64/chromedriver.exe"
```
Comment (VAlexandrov911) [inline module_13/tests/base_test.py:7]: Just call the default constructor to use Chromedriver by
driver = webdriver.Chrome()

PS: don't forget to add chromedriver's path to PATH env variables
or try to use ChromeDriverManager (https://pypi.org/project/webdriver-manager/)

File: module_13/tests/login_test.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+from selenium import webdriver
+from module_13.pages.login_page import LoginPage
+
+
+def test_login():
+    chrome_driver_path = "../driver/chromedriver-win64/chromedriver.exe"
```
Comment (VAlexandrov911) [inline module_13/tests/login_test.py:6]: regarding your question in private messages, you can use your setup and teardown functions in few different ways in your frameworks, for example in pytest you can use fixtures
https://docs.pytest.org/en/6.2.x/fixture.html

PS: you will get all info about it in Pytest lecture btw

**Review by VAlexandrov911 — APPROVED** [review]
Actually, from my side regarding Selenium base methods and locators usage everything is ok,
just try to avoid paths to webdriver in your code (I've provided few usefull links in comments).
So I will approve it for now, GJ!

File: module_13/driver/chromedriver-win64/chromedriver.exe
Comment (sergei820) [reply module_13/driver/chromedriver-win64/chromedriver.exe:1]: Thank you a lot, Vasilii!

File: module_13/tests/login_test.py (hunk 2 above)
Comment (sergei820) [reply module_13/tests/login_test.py:6]: Thank you! We just passed fixtures in the text module, with fixtures it's not a problem anymore :)

**Pull request merged**